Dockerfile

```diff
@@ -17,17 +17,17 @@
 # Now you can connect to:
 #    http://172.17.0.1:8891
 #
-FROM ubuntu:14.04
+FROM ubuntu:16.04
 MAINTAINER pietro.delsante_AT_gmail.com
 ENV DEBIAN_FRONTEND noninteractive
 EXPOSE 8891
 
 RUN apt-get update && \
-    apt-get -y install curl && \
+    apt-get -y install curl sudo && \
     curl -sL https://deb.nodesource.com/setup_4.x | sudo bash - && \
     apt-get -y install nodejs build-essential xvfb libfontconfig1 && \
-    npm install -g slimerjs@0.9.6-2 && \
-    npm install -g phantomjs@1.9.19 && \
-    npm install -g manet@0.4.10
+    npm install -g slimerjs@0.906.2 && \
```

Comment:
Yes. 0.10 need to download from source code. I tried "npm install slimerjs" that did not indicate versions, and checked the latest version of npm package.

Dominik Deren tried to upgrade slimerjs 0.10. But I did not try the code. This is because that I think npm will support it.
see: https://github.com/import-io/manet-dockerfile/blob/197e911b407719100c7ccf37fc4e4fd00e1cd8e5/Dockerfile